examples/using-add-filestore/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"gofr.dev/pkg/gofr"
+	"gofr.dev/pkg/gofr/datasource/file"
+	"gofr.dev/pkg/gofr/datasource/file/ftp"
+	"gofr.dev/pkg/gofr/logging"
+)
+
+type FileServerType int
+
+const (
+	FTP FileServerType = iota
+	SFTP
+)
+
+// This can be a common function to configure both FTP and SFTP server.
+func configureFileServer(app *gofr.App) file.FileSystemProvider {
+	port, _ := strconv.Atoi(app.Config.Get("PORT"))
+
+	return ftp.New(&ftp.Config{
+		Host:      app.Config.Get("HOST"),
+		User:      app.Config.Get("USER_NAME"),
+		Password:  app.Config.Get("PASSWORD"),
+		Port:      port,
+		RemoteDir: app.Config.Get("REMOTE_DIR_PATH"),
+	})
+}
+
+func printFiles(files []file.FileInfo, err error) {
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Can this error be handled at the calling statement? It is so because the name of the function suggests that it is intended to print files (names) hence it should not be handling error especially when error in coming as the function argument.